### ISSUE TITLE
automate AMI updates

### ISSUE THREAD

**Issue by errordeveloper** [weaveworks/eksctl#49]
Currently, AMI IDs have to copied from [the docs](https://docs.aws.amazon.com/eks/latest/userguide/launch-workers.html), we need to make it easier to automate (e.g. via a script).

**Comment by JordanFaust** [comment]
Commenting here rather then creating a new issue. As part of this the cli should expose an option to create a cluster with a custom AMI for the worker nodes. Thoughts on rolling these two things together?

**Comment by errordeveloper** [comment]
Custom AMI will be supported soon, we just need to expose a flag, I'd like to do that as part of #48 (there is code already, but I'd like to simplify the code by using a library, if I can).

**Comment by errordeveloper** [comment]
```
aws ec2 describe-images \
    --filters \
        Name=owner-id,Values=602401143452 \
        Name=virtualization-type,Values=hvm \
        Name=root-device-type,Values=ebs \
        Name=is-public,Values=true \
        Name=name,Values=amazon-eks-node-\* \
    --output text \
    --query 'max_by(Images[], &CreationDate).ImageId'
```

**Comment by richardcase** [comment]
I'm making this change as part of #192 as you suggested.

**Comment by richardcase** [comment]
@errordeveloper - do you think that this can be closed now (after #221)? Or would you like to wait until we get an automated test that checks for every evening for new image versions?

**Comment by errordeveloper** [comment]
@richardcase yes :)
